plugins/home/src/components/HomepageCompositionRoot.tsx

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2021 The Backstage Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { ReactNode } from 'react';
+import { useOutlet } from 'react-router';
+import { Content, Header, Page } from '@backstage/core-components';
+
+export const HomepageCompositionRoot = (props: {
+  title?: string;
+  children?: ReactNode;
+}) => {
+  const outlet = useOutlet();
+  const children = props.children ?? outlet;
+  return (
+    <Page themeId="home">
+      <Header title={props.title ?? 'Home'} />
```

Comment:
not sure if we use lifecycle labels anymore but wonder if we should make it clear somewhere what state this plugin is in, either in the documentation or in the header with a [lifecycle label](https://backstage.io/storybook/?path=/story/feedback-lifecycle--alpha-default).

Reply:
Right! We removed the Lifecycle label since it could be confusing to find an alpha tag on the home page, as pointed out in a previous comment in this PR. Perhaps the README would be a good place?